### PR TITLE
Make Demo Not Terrible

### PR DESCRIPTION
**Pull request by vermont42**

### CODE REVIEW

File: Demo/AppDelegate.swift [hunk 1]
```diff
@@ -13,34 +13,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+    window?.rootViewController = demoVC
+    window?.makeKeyAndVisible()
     return true
   }
 
-  func applicationWillResignActive(_ application: UIApplication) {
-    // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
-    // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
-  }
+  func applicationWillResignActive(_ application: UIApplication) {}
```
Comment (nkanetka) [inline Demo/AppDelegate.swift]: I'd just get rid of these methods entirely...No point in keeping them unless we make use of them

File: Demo/DemoView.swift [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  required init(coder aDecoder: NSCoder) {
+    fatalError("This class does not support NSCoding.")
+  }
+
+  internal func updateSpeedLabel(speed: Double) {
+    speedLabel.text = "\(Int(speed))X"
+  }
+
+  internal func enableSpeedControls() {
+    [speedLabelLabel, speedLabel, speedStepper].forEach {
```
Comment (nkanetka) [inline Demo/DemoView.swift]: These should be class variables so we don't have copy and paste code

Comment (nkanetka) [reply Demo/DemoView.swift]: Disregard, I pulled the files and see what you are doing. I'm good with the way it is :)

**Comment by vermont42** [comment]
Thanks for the review. I have nothing else to add, so I'll just merge.

**Pull request merged**